source/_includes/common-tasks/enable_i2c.md

```diff
@@ -32,19 +34,17 @@ and make sure the first partition is available.
   dtparam=i2c_arm=on
   ```
 
-#### Step 3 - Start with the new configuration
+#### Use SD Card/Step 3 - Start with the new OS configuration
 
 - Insert the SD card back into your Raspberry Pi.
 - On startup, the `hassos-config.service` will automatically pickup the new
   `rpi-i2c.conf` configuration.
 - Another reboot might be necessary to make sure the just imported `rpi-i2c.conf` is
   present at boot time.
 
-The I2C devices should now be present under /dev.
+### Alternative to SD Card via Home Assistant Operating System Terminal
```

Comment:
Could you explain why this title needed changing?

Reply:
Shortend the section title